src/spikeinterface/extractors/tests/test_nwb_s3_extractor.py

```diff
@@ -36,7 +37,7 @@ def test_s3_nwb_ros3():
         trace_scaled = rec.get_traces(segment_index=segment_index, return_scaled=True, end_frame=2)
         assert trace_scaled.dtype == 'float32'
 
-
+@pytest.mark.streaming_extractors
```

Comment:
@h-mayorquin are markers mutually exclusive? If not, we should make sure that in the `conftest.py` we skip this file when marking `extractors`

Reply:
Check the other comment.